src/services/simulationService.ts

```diff
@@ -84,9 +84,9 @@ class simulationService {
     }
     switch (zone.zoneType) {
       case ZoneType.Video:
-        return new VideoSimulator(zone.id, zone.params, renewable);
```

Comment:
if we change to give zone as first parameters, we can probably remove the seconds parameters which is zone.params

Reply:
You're right, thanks. I'll change it.